packages/block-editor/src/components/media-upload/index.native.js

```diff
@@ -286,4 +304,12 @@ export class MediaUpload extends Component {
 	}
 }
 
-export default MediaUpload;
+export default compose( [
+	withSelect( ( select ) => {
+		return {
+			isAudioBlockEnabled:
```

Comment:
Should I rename`isAudioBlockEnabled` to `isMediaUploadEnabled`? Most capabilities are named after their blocks but I think for this situation the naming paradigm could be different because we are not enabling/disabling the block itself. We are just disabling some of it's functionality. Let me know if you have any thoughts.

Reply:
+1 one having the names as close as possible to the actual actions performed. The block will be enabled always and it's only some subfeatures that will be gated.

Reply:
>+1 on having the names as close as possible to the actual actions performed. The block will be enabled always and it's only some subfeatures that will be gated.

I agree on this, it could be a bit misleading using `isAudioBlockEnabled` as it's not really enabling the block but the media upload.